api/types/types.go

```diff
@@ -249,17 +249,19 @@ type ExecStartCheck struct {
 // ContainerState stores container's running state
 // it's part of ContainerJSONBase and will return by "inspect" command
 type ContainerState struct {
-	Status     string
-	Running    bool
-	Paused     bool
-	Restarting bool
-	OOMKilled  bool
-	Dead       bool
-	Pid        int
-	ExitCode   int
-	Error      string
-	StartedAt  string
-	FinishedAt string
+	Status         string
```

Comment:
Maybe. Is that not also true of Running, Paused, Restarting though? I have no opinion here, was just trying to mirror what was already there.